src/components/tabs/tabs.test.js

```diff
@@ -296,33 +284,37 @@ context("Testing Tabs component", () => {
     });
 
     it.each([
-      ["tab-5", "Content for tab ", 5],
-      ["tab-4", "Content for tab ", 4],
-      ["tab-3", "Content for tab ", 3],
-      ["tab-2", "Content for tab ", 2],
-      ["tab-1", "Content for tab ", 1],
+      ["tab-5", 5],
```

Comment:
same here with `tab-5` 
so you will be able to use:


CypressMountWithProviders(<TabsComponent selectedTabId={{`tab-${id}`}} />);

Reply:
Had to change slightly to selectedTabId={`tab-${id}`}